### PR TITLE
Release 0.2

### PR DESCRIPTION
**Pull request by RaitoBezarius**
:tada: :tada:

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-pierre <pierre.vigier@ymail.com>
+Luc Rocher
+Pierre Vigier
 Ryan Lahfa <masterancpp@gmail.com>
-Soubi <camille.poquet@gmail.com>
-vendethiel <vendethiel@hotmail.fr>
-Xia Li-yao <Lysxia@users.noreply.github.com>
-Étienne Simon <esimon@esimon.eu>
+vendethiel
+Xia Li-yao
```
Comment (jilljenn) [inline AUTHORS]: C'est dans l'autre sens, le nom-prénom.

**Comment by jilljenn** [comment]
Okay actually you should:

- Change the tweet towards https://twitter.com/MangakiFR/status/879357016925511682
- Put a link towards your blog post! (So you should start by the blog post.)

**Comment by RaitoBezarius** [comment]
OK, we have this result: http://screencloud.net/v/4n3ZB

Was this what you expected?

**Comment by jilljenn** [comment]
Oh you did not commit yet! Please just add “Blog :” before « Pourquoi vous devriez » please! Then perfect :)

**Comment by jilljenn** [comment]
Or actually, please switch both blog posts. Kyoto University should be a line and the new features should be a Twittercard.

**Pull request merged**